clients/drcachesim/simulator/cache_simulator.cpp

```diff
@@ -440,9 +440,6 @@ cache_simulator_t::process_memref(const memref_t &memref)
         return true;
     }
 
-    // We use a static scheduling of threads to cores, as it is
-    // not practical to measure which core each thread actually
-    // ran on for each memref.
     int core;
     if (memref.data.tid == last_thread_)
```

Comment:
Is this correct? There could be an intervening TRACE_MARKER_TYPE_CPU_ID that changed the core for last_thread_, right?

Reply:
Hmm, you're saying it's not clear last_thread_ on a cpu marker?  Added.